recipe/install_nvcc.sh

```diff
@@ -124,6 +124,11 @@ EOF
 mkdir -p "${PREFIX}/bin"
 cat > "${PREFIX}/bin/nvcc" <<EOF
 #!/bin/bash
-"\${CUDA_HOME}/bin/nvcc" -ccbin "\${CXX}" \$@
+for i ; do
+  case \$i in -ccbin)
+    exec "\${CUDA_HOME}/bin/nvcc" "\${@}"
+  esac
+done
```

Comment:
Would it make sense to just loop through `${@}` and see if `-ccbin` is there without the `exec`? Might simplify things a bit 🙂

Reply:
> Would it make sense to just loop through `${@}`

This is exactly what `for i` does ;)
> without the `exec`?

The `exec` is just an independent improvement: If we have a launcher script, it makes sense to `exec` the called process so that our shell process can be stopped/replaced.

Reply:
Interesting would have expected that to look more like `for i in "${@}"`. It might be clearer to write that (even if `bash` is doing some magic under-the-hood).

Reply:
> (even if `bash` is doing some magic under-the-hood).

It's standard POSIX shell: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_04_03

> Interesting would have expected that to look more like `for i in "${@}"`. It might be clearer to write that

I don't mind either way -- I can add a commit to change that :).

Reply:
It would make it clearer I think. We already are `cat`ing into a script (my fault 😞). So it's already a bit confusing. The less confusing we can make it the better 🙂